### PR TITLE
Move "delete record" feature from the admin settings page to a separated page.

### PR DESCRIPTION
**Pull request by gimite**
It looks weird to have "delete record" feature in the settings page, especially with the new admin pages structure with the side bar.

### CODE REVIEW

File: app/resources/admin_delete_record.html.template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+{% extends "admin-base.html.template" %}
+{% load i18n %}
+
+{% block content %}
+  <h2>Delete a record</h2>
+
+  <form method="post" class="admin">
+    <input type=hidden name=operation value=delete>
+    person_record_id: <input name="id" id="id" size=50 value="{{id}}">
```
Comment (kevinalle) [inline app/resources/admin_delete_record.html.template]: size="50"
for valid HTML

File: app/resources/admin_delete_record.html.template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{# (see admin_delete_record.py). #}
+
+{% extends "admin-base.html.template" %}
+{% load i18n %}
+
+{% block content %}
+  <h2>Delete a record</h2>
+
+  <form method="post" class="admin">
+    <input type=hidden name=operation value=delete>
```
Comment (kevinalle) [inline app/resources/admin_delete_record.html.template]: <input type="hidden" name="operation" value="delete">

**Comment by kevinalle** [comment]
minor comments, LGTM

File: app/resources/admin_delete_record.html.template (hunk 1 above)
Comment (gimite) [inline app/resources/admin_delete_record.html.template]: Done.

File: app/resources/admin_delete_record.html.template (hunk 2 above)
Comment (gimite) [inline app/resources/admin_delete_record.html.template]: Done.

**Comment by gimite** [comment]
Thanks! Merged.

**Pull request merged**